Core/include/Acts/Material/Material.hpp

```diff
@@ -111,12 +117,6 @@ class Material {
   float m_ar = 0.0f;
   float m_z = 0.0f;
   float m_molarRho = 0.0f;
-
-  friend constexpr bool operator==(const Material& lhs, const Material& rhs) {
-    return (lhs.m_x0 == rhs.m_x0) && (lhs.m_l0 == rhs.m_l0) &&
-           (lhs.m_ar == rhs.m_ar) && (lhs.m_z == rhs.m_z) &&
-           (lhs.m_molarRho == rhs.m_molarRho);
-  }
```

Comment:
This is generally preferred over member functions, because it's symmetric and does not participate in general overload resolution.